sound/soc/sof/core.c

```diff
@@ -340,9 +340,17 @@ EXPORT_SYMBOL(snd_sof_device_probe);
 
```

Comment:
typo in commit: controler  -> controller

Reply:
Yeah, thanks for spotting it.